sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -131,13 +130,12 @@ export class PartitionPump {
           lastSeenSequenceNumber = receivedEvents[receivedEvents.length - 1].sequenceNumber;
         }
 
-        const span = createProcessingSpan(
-          receivedEvents,
-          this._context.config,
-          this._processorOptions
+        await tracingClient.withSpan(
+          "PartitionPump.process",
+          {},
```

Comment:
no need to pass the operation options?

Reply:
CommonEventProcessorOptions doesn't have `tracingOptions` on it, does it?

Reply:
It does! Not sure what it actually is used for but 
![image](https://user-images.githubusercontent.com/753570/153686693-afbc6d35-c8b1-4564-98e9-5f5f3a1eedcc.png)

It looks like it extends SubscribeOptions which does have tracingOptions... I just don't know if those are the "correct" tracing options...

Reply:
processEvents is already being traced.

I don't think we'll get anything much from this tracingOptions, perhaps it's just an over-exposed option in the SubscribeOptions, or at least seems not relevant in here.

Question: Do you expect this `tracingContext` to be useful for the users while processing the events?
https://github.com/Azure/azure-sdk-for-js/blob/c922093bc8680a2616c7f40073b9fed265bb2363/sdk/core/core-tracing/src/interfaces.ts#L277-L280

Reply:
> processEvents is already being traced.
> 
> I don't think we'll get anything much from this tracingOptions, perhaps it's just an over-exposed option in the SubscribeOptions, or at least seems not relevant in here.


Makes sense, I will pass an empty options object for now. We can always change it later.

> 
> Question: Do you expect this `tracingContext` to be useful for the users while processing the events?
> 
> https://github.com/Azure/azure-sdk-for-js/blob/c922093bc8680a2616c7f40073b9fed265bb2363/sdk/core/core-tracing/src/interfaces.ts#L277-L280

Unclear right now - most users will just grab the active context from OTel via `context.active()` which we do set as part of `withSpan`. For customers who need an explicit context we can always pass it in (but, and not assuming this is the direction you were headed, we can always pass it through explicitly)